joblib/__init__.py

```diff
@@ -124,8 +124,7 @@
 from .parallel import parallel_backend
 from .parallel import effective_n_jobs
 
-from .externals.loky import wrap_non_picklable_objects
-
+from .externals.cloudpickle_wrapper import wrap_non_picklable_objects
```

Comment:
This is problematic. The loky vendoring script will not work anymore and users would expect loky consistency will be lose.

I would rather wrap this with a try / except ImportError and not export the `wrap_non_picklable_objects` in `__all__` when loky cannot be imported. Add a comment to explain that this can happen on systems that do not support sem_open (e.g. Android).